### ISSUE TITLE
Citations is featured in GNOME Circle, but not translatable via Damned Lies

### ISSUE THREAD

**Issue by daPhipz** [cassidyjames/clairvoyant#52]
Hello!
This is a fun app you got there xD

I noticed that, while Clairvoyant is featured in [circle.gnome.org](circle.gnome.org), it is not translatable via GNOME's translation infrastructure over at [l10n.gnome.org](l10n.gnome.org). Was this a deliberate decision?

**Comment by cassidyjames** [comment]
@daPhipz no, I just haven't gotten set up with the translation infra. I'll look into it, thanks!

**Comment by daPhipz** [comment]
Cool, nice to know. I'm definitely ready to start translating ;)
